guide/process/process-file.md

```diff
@@ -156,10 +196,65 @@ All conditions should match to continue to the next step.
   }]
 }" :types="{}" />
 
+#### Example
+```yaml
+key: erc20-notification
+steps:
+  - type: trigger
+    instance:
```

Comment:
One example with the `instanceHash` would be great too like that we have the 3 different versions